recipes/libjaylink/recipe.yaml

```diff
@@ -0,0 +1,118 @@
+context:
+  name: libjaylink
+  version: "0.4.0"
+  build_num: 0
+  so_version: "0.2.1"
+  major_so_version: "0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://gitlab.zapb.de/libjaylink/libjaylink/-/archive/${{ version }}/libjaylink-${{ version }}.tar.gz
+  sha256: 5557d623934a4bbc053c11f9a181375d7abeb76af910696d9e3d9b1de3bf6987
+  patches:
+    - if: win
+      then:
+        - patches/add-jaylink.def.patch
+        - patches/add-jaylink.lib-meson.build.patch
+        - patches/enable-dynamic-library.patch
+        - patches/fix-types-discovery_tcp.c.patch
+        - patches/fix-types-discovery_usb.c.patch
+        - patches/fix-types-fileio.c.patch
+        - patches/fix-types-socket.c.patch
+        - patches/fix-types-strutil.c.patch
+        - patches/fix-types-transport_tcp.c.patch
+        - patches/fix-types-transport_usb.c.patch
+        - patches/fix-uninitialized.patch
+
+build:
+  number: ${{ build_num }}
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ninja
+    - pkg-config
+    - if: win
+      then:
+        - m2w64-binutils
+  host:
+    - libusb
+    - meson >=1.1.0
+  run_exports:
+    - ${{ pin_subpackage('libjaylink', upper_bound='x.x') }}
```

Comment:
Nice work! My only comment is that you might want to ask upstream when you can relax this pinning. Considering that they version the ABI and API separately, this might be too strict!

Reply:
Appreciated. Will do thank you